test/functional/test_solution.py

```diff
@@ -0,0 +1,439 @@
+# Copyright 2106 F5 Networks Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import decorator
+import paramiko
+import pytest
+import time
+import sys
+
+from f5.bigip import BigIP
+from pprint import pprint as pp
+from neutronclient.v2_0 import client
```

Comment:
Is there a different library for LBaaSv2 or are they in the same library?  How do you tell it to use the various versions?